platform-commands/pom.xml

```diff
@@ -31,7 +31,7 @@
         <dependency>
             <groupId>org.apache.karaf.shell</groupId>
             <artifactId>org.apache.karaf.shell.console</artifactId>
-            <version>2.3.2</version>
```

Comment:
I like the use of the property as it will help maintain the Karaf version. Leaving older versions can lead to false positives in the OWASP scans as well.